### PR TITLE
Use abundances, densities filepaths relative to w7 config file.

### PR DESCRIPTION
**Pull request by karandesai-96**
- Names of abundances and densities profile files have been corrected in config file of W7 setup. (Directory: **tardis/tests/tests_slow/w7**)

- Changes made in setup method of TestW7 class: instead of reading yaml to dict and forming a config, the config is directly formed by file path of yaml file. This ensures that relative filepaths are taken care of. 

This amendment is exercised as per proposal of @lukeshingles in #543 .

### CODE REVIEW

File: tardis/tests/tests_slow/test_w7.py [hunk 1]
```diff
@@ -49,7 +39,8 @@ def setup(self, request, reference, data_path, atomic_data_fname,
         assert self.atom_data.uuid1 == kurucz_data_file_uuid1
 
         # Create a Configuration through yaml file and atom data.
-        tardis_config = Configuration.from_config_dict(config_yaml, self.atom_data)
+        tardis_config = Configuration.from_config_dict(
```
Comment (karandesai-96) [inline tardis/tests/tests_slow/test_w7.py]: It didn't happen. It raised an `OSError` of file doesn't exist.

Comment (karandesai-96) [inline tardis/tests/tests_slow/test_w7.py]: @yeganer ... as it didn't work, I took reference from [this](https://github.com/tardis-sn/tardis/pull/543/files#diff-36f090313b4e1f6a024f4cbc0537a305R42) line.

Comment (yeganer) [inline tardis/tests/tests_slow/test_w7.py]: Can you please post the error? From my understanding it should be possible to use Configuration.from_yaml and that uses the relative paths automatically. I even patched `.from_yaml` to allow atom_data to be passed along

Comment (karandesai-96) [inline tardis/tests/tests_slow/test_w7.py]: I'm sorry, I misread your comment as `from_config_dict`. I'll run it with `from_yaml`

**Comment by karandesai-96** [comment]
@yeganer This PR works properly now. :white_check_mark:

**Comment by yeganer** [comment]
When using `git commit --amend` you should check that the commit message still reflects the changes done by this commit.
Currently there are some leftovers from the perevious one:

>  Changes made in setup method of TestW7 class:
  while loading the config from the config dict,
  an extra kwarg 'config_dirname' is passed.

As travis doesn't run these tests, can you please upload a dokureport with the latest commit?

**Comment by yeganer** [comment]
@tardis-sn/tardis-core Feel free to merge at will.

**Pull request merged**